doc/examples/features_detection/plot_sift.py

```diff
@@ -22,16 +22,16 @@
 descriptor_extractor = SIFT()
 
 descriptor_extractor.detect_and_extract(img1)
-keypoints1 = descriptor_extractor.keypoints
-descriptors1 = descriptor_extractor.descriptors
+keypoints1 = descriptor_extractor.keypoints[:100]
+descriptors1 = descriptor_extractor.descriptors[:100]
```

Comment:
I think I would define a variable, something like `num_keep = 100` and add a comment above this noting that we have restricted the number of keypoints and matches for clarity in the plots

My only concern is that this could be misleading in terms of how many features SIFT typically finds. Maybe it is worth keeping the plot with all of the features first and then creating a second figure where the features are limited to a subset of 100.

Reply:
![myplot](https://user-images.githubusercontent.com/56137202/126072367-246da16c-f5eb-42c6-b12c-a67882fe0fe3.png)
@grlee77 Would you prefer it like this?